From 1c644eba660e7928d5099829e86c2ebf1b974e19 Mon Sep 17 00:00:00 2001
From: Junhao He <hejunhao3@huawei.com>
Date: Wed, 4 Jan 2023 19:08:41 +0800
Subject: [PATCH] perf: hisi: Fix read sccl_id and ccl_id error in TSV200
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 1c644eba660e7928d5099829e86c2ebf1b974e19
Modified-by-SEL: No


driver inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I68F5A
CVE: NA

--------------------------------

In TSV200 platform, SCCL is Aff3[7:0], CCL is Aff2[7:0] if mt.
Fixes: 1af26389a3 ("perf: hisi: Fix read sccl_id and ccl_id error in some platform")

Signed-off-by: Junhao He <hejunhao3@huawei.com>
Reviewed-by: Yang Shen <shenyang39@huawei.com>
Reviewed-by: Yang Jihong <yangjihong1@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/perf/hisilicon/hisi_uncore_pmu.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/perf/hisilicon/hisi_uncore_pmu.c b/drivers/perf/hisilicon/hisi_uncore_pmu.c
index 66ed28c2c544..9dbd3a98f7ad 100644
--- a/drivers/perf/hisilicon/hisi_uncore_pmu.c
+++ b/drivers/perf/hisilicon/hisi_uncore_pmu.c
@@ -437,7 +437,6 @@ static void hisi_read_sccl_and_ccl_id(int *scclp, int *cclp)
 	if (mt) {
 		switch (read_cpuid_part_number()) {
 		case HISI_CPU_PART_TSV110:
-		case HISI_CPU_PART_TSV200:
 		case ARM_CPU_PART_CORTEX_A55:
 			sccl = aff2 >> 3;
 			ccl = aff2 & 0x7;
-- 
2.33.0

